Content.Server/Botany/Components/PlantHolderComponent.cs

```diff
@@ -71,6 +71,9 @@ public sealed partial class PlantHolderComponent : Component
     [ViewVariables(VVAccess.ReadWrite), DataField("health")]
     public float Health;
 
+    [ViewVariables(VVAccess.ReadWrite), DataField("potencyBonus")]
+    public float PotencyBonus;
```

Comment:
Please document why this is here and how it is different from `Potency` in SeedData.